imports/voting/ui/particles/SignStatement.vue

```diff
@@ -157,7 +163,7 @@
   .voting-section {
     &.doubt {
       .tab {
-        background: $red;
+        background: $red !important;
```

Comment:
I believe that there is another way without using of `!Important`.

Reply:
Which way man ?